<launch>

  <node pkg="erp42_driver" type="erp42_main" name="erp42_main">
    <param name="port" value="/dev/erp42"/>
    <param name="frequency" value="20"/>
    <param name="wheelbase" value="1.12"/>
    <param name="maximum_steering_angle" value="2000"/>
    <param name="minimum_turning_radius" value="2.6"/>
    <param name="initWriteThread" value="true"/>
	
		<param name="settling_time" value="1.7"/>
		<param name="weight" value="1.1"/>
		<param name="shift" value="20.0"/>
		<param name="steer_kp" value="1.0"/>
		<param name="steer_ki" value="0.0"/>
		<param name="steer_kd" value="0.0"/>
		<param name="brake_max" value="200"/>
		<param name="brake_kp" value="25"/>
		<param name="brake_ki" value="0.0"/>
		<param name="brake_kd" value="0.0"/>
		<param name="accel_kp" value="1.0"/>
		<param name="accel_ki" value="0.5"/>
		<param name="accel_kd" value="0.0"/>
			
		<param name="C0" value="0.04"/> <!-- original value 0.108 -->
		<param name="C1" value="0.005"/>
		<param name="C2" value="0.0598"/>

    	<param name="max_estop_time" value="30"/>
	
	<!-- <rosparam file="$(find erp42_driver)/cfg/lowlevel_controller.yaml" command="load" /> -->

  </node>

  <!-- <node pkg="rqt_plot" name="rqt_plot" type="rqt_plot"/> -->

</launch>
